python/lsst/ts/externalscripts/make_LOVE_stress_tests.py

```diff
@@ -0,0 +1,299 @@
+            an `lsst.ts.salobj.Remote`
+
+        Constructing a `salobj.Remote` is slow (DM-17904), so configuration
+        may take a 10s or 100s of seconds per CSC.
+        """
+        self.log.info("Configure started")
+
+        # set configurations
+        self.location = config.location
+        self.username = config.username
+        self.password = config.password
+
+        # construct remotes
+        remotes = dict()
+        for elt in config.data:
+            name, index = salobj.name_to_name_index(elt)
+            self.log.debug(f"Create remote {name}:{index}")
+            if elt not in remotes:
+                remote = salobj.Remote(domain=self.domain, name=name, index=index)
+                remotes[elt] = remote
+
+        self.remotes = remotes
+
+    async def run(self):
+        """Run script."""
+        tasks = [
+            remote.start_task
+            for remote in self.remotes.values()
+            if not remote.start_task.done()
+        ]
+        if tasks:
+            self.log.info(f"Waiting for {len(tasks)} remotes to be ready")
+            await asyncio.gather(*tasks)
+
+        # Enable all CSCs
+        for csc, remote in self.remotes.items():
+            evt = await remote.evt_summaryState.aget()
+            state = salobj.State(evt.summaryState)
+
+            if state == salobj.State.STANDBY:
+                await remote.cmd_start.start(timeout=self.cmd_timeout)
+                await remote.cmd_enable.start(timeout=self.cmd_timeout)
+
+        event_streams = dict()
+        telemetry_streams = dict()
+        for producer in self.remotes.keys():
+            event_streams[producer] = salobj.SalInfo(
+                self.domain, producer.split(":")[0]
+            ).__getattribute__("event_names")
+            telemetry_streams[producer] = salobj.SalInfo(
+                self.domain, producer.split(":")[0]
+            ).__getattribute__("telemetry_names")
+
+        # Create clients and listen to ws messages
+        loop = asyncio.get_event_loop()
+        for i in range(self.number_of_clients):
+            self.clients.append(
+                ClientInstance(
+                    self.location,
+                    self.username,
+                    self.password,
+                    event_streams,
+                    telemetry_streams,
+                )
+            )
+
+        for client in self.clients:
+            client.request_token()
+            loop.create_task(client.start_ws_client())
+
+    async def clean(self):
+        """Return the system to its default status."""
+
+        for csc, remote in self.remotes.items():
+            evt = await remote.evt_summaryState.aget()
+            state = salobj.State(evt.summaryState)
+
+            if state == salobj.State.ENABLED:
+                await remote.cmd_disable.start(timeout=self.cmd_timeout)
+                await remote.cmd_standby.start(timeout=self.cmd_timeout)
```

Comment:
Consider replacing this entire loop body with`await salobj.set_summary_state(remote=remote, state=salobj.State.STANDBY)`. It is simpler and works from any initial state except FAULT.